### DISCUSSION TITLE
Installing according to instuction and getting .dev version

### DISCUSSION THREAD

**Discussion by ESP4Ever** [Chia-Network/chia-blockchain#8278] [Support]
Hi
Doing installation as written in wiki and always getting "1.2X+1.devY" version (1.2.6.dev0 currtently)
....
git clone https://github.com/Chia-Network/chia-blockchain.git -b latest --recurse-submodules 
...
...
> chia version
1.2.6.dev0

Can you please explain what's wrong?

**Comment by loppefaaret** [comment]
it is most likely leftovers of changed files in the cloned directories - the .+1dev0 is a direct mirror of the previous release, is is functionally identical to the previous release version, apart from reporting version as .+1dev0
in other words, is it purely cosmetic - it has no impact of the function

**Comment by BrandtH22** [comment]
@ESP4Ever , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
